### ISSUE TITLE
Waardebereik van metadata die niet een normale string is

### ISSUE THREAD

**Issue by lennartvanbergen-kadaster** [Geonovum/MIM-Werkomgeving#162]
Bv. lengte, dit is niet characterstring, maar characterstring met als toegestane waarden: een integer, een ',' en 'een ..' 
Bv. kardinaliteit is niet characterstring, maar characterstring met als toegestane waarden: een integer, een '..' en  een '* '
Er zijn links en rechts nog wat van dit soort kleine dingetjes te vinden. Dit komt naar voren als we een informatiemodel in MIM taal exporteren naar bv. XML of JSON. In de XMI is deze data wel aanwezig, en in de MIM documentatie ook, het is alleen niet aan elkaar gedefinieerd bij het metadata aspect.

**Comment by lennartvanbergen-kadaster** [comment]
Bv. aggregatietype van een relatie is een enum.

**Comment by PalmJanssen** [comment]
Het is nodig als je het UML van het metamodel wilt gebruiken om een XML schema te genereren, of een andere serialisatie van het metamodel.

**Comment by PalmJanssen** [comment]
Hoort in nieuwe bijlage waarin MIM - UML uitgedrukt in MIM. Op basis van dat document kan je MIM producten afleiden. (bijvoorbeeld XSD).

**Comment by lennartvanbergen-kadaster** [comment]
Klopt, is wel nodig voor MIM 1.1.1. release, als randvoorwaarde om MIM-export formaat te maken, moet dus echt mee met 1.1.1.
